examples/hello/bpSZWrapper/helloBPSZWrapper.cpp

```diff
@@ -9,7 +9,6 @@
  *  Created on: Jul 26, 2017
  *      Author: William F Godoy godoywf@ornl.gov
  */
-#include <mpi.h>
```

Comment:
Keep the mpi.h include, just move it to after adios2.h and guard with the ifdef.  Otherwise everything else looks good.

Reply:
I can definitely do that, but I removed it because we have tests with MPI where we don't include it.  And it worked without it.

Reply:
It works because it's getting pulled in transitively by adios2.h but we shouldn't expect or rely on transitive includes.  If we explicitly make MPI calls then we should explicitly include mpi.h.  The other tests should be changed at some point.

Thanks for fixing this!